MapboxNavigation/Style.swift

```diff
@@ -237,6 +237,11 @@ open class Style: NSObject {
     public var styleType: StyleType = .daytimeStyle
     
     /**
+     Map style to be used for the style.
+     */
+    public var mapStyle: URL?
```

Comment:
Nit: this name indicates that it's a style which it isn't. Can we rename to `mapStyleURL`?